### PR TITLE
Minutes tech lead call 24 October 2023

### PR DESCRIPTION
**Pull request by valeriupredoi**
For your consideration @ESMValGroup/technical-lead-development-team :beer:

### CODE REVIEW

File: Tech Lead/Minutes/20231024.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+|2023.07|@bouweandela, @remi-kazeroni|Asking the community to check the output once for all| @ehogan: see [ESMValTool: discussion: #3247](https://github.com/ESMValGroup/ESMValTool/discussions/3247) for some additional thoughts on this|
+|2023.07|@valeriupredoi|Shared gmail account to handle to Zenodo credentials|pending|
+|2023.09|@schlunma|Talk about ERA5|pending|
+
+## Topics Discussed
+
+- @zklaus: release situation for 2.10
+  - things are a bit more difficult this time round (new names for obs4MIPS, changes to recipe handling) - @bouweandela fixed a couple issues; further help is needed
+  - Some files are not found in the ``download`` directory if the ESGF node where the data is from is down
+  - @bouweandela (and us) agrees we should get the ESMValCore relase candidate out anyway - we'll test all recipes when CEDA comes bac online
```
Comment (bouweandela) [inline Tech Lead/Minutes/20231024.md:30]:
```suggestion
  - @bouweandela (and us) agrees we should get the ESMValCore release candidate out anyway - we'll test all recipes when CEDA comes back online
```

File: Tech Lead/Minutes/20231024.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  - PRs to change obs4MIPS name changes (by Romain) - @bouweandela mentions we should merge them to get the data finding to be using modern names
+  - a couple bottom lines: we'll test everything that don't need CEDA, and we'll get Core RC out
+- @bouweandela: ESMValBot and the VM (former: Remi): we need one maintainer for these: V @valeriupredoi volunteers, Bouwe will meet V at some point talk about it;
+- Mambafore: we'll stick with it for now, until major problems appaear (if any)
```
Comment (bouweandela) [inline Tech Lead/Minutes/20231024.md:34]:
```suggestion
- Mambafore: we'll stick with it for now, until major problems appear (if any)
```

**Review by bouweandela — APPROVED** [review]
Looks good, but would recommend using an automatic spell checker ;-)

File: Tech Lead/Minutes/20231024.md (hunk 2 above)
Comment (valeriupredoi) [reply Tech Lead/Minutes/20231024.md:34]: haa! You missed one too Mambafore -> Mambaforge :rofl:

**Comment by valeriupredoi** [comment]
what are you saying @bouweandela - BAC is a perfectly fine word: https://en.wikipedia.org/wiki/British_Aircraft_Corporation :laughing: Cheers for the review, bud :beer:

**Pull request merged**